src/validate.js

```diff
@@ -0,0 +1,50 @@
+const validCases = ['SNAKE', 'CAMEL', 'KEBAB', 'PASCAL', 'UPPER'];
+
+const noTextErrorMessage =
+  'Text to convert is required. ' +
+  'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+
+const noParamsErrorMessage =
+  '"toCase" query param is required. ' +
+  'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+
+const invalidCaseErrorMessage =
+  'This case is not supported. ' +
+  'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.';
```

Comment:
would be great to combine the error messages into an object and move all constants to a separate file